app/graphs.py

```diff
@@ -1,5 +1,17 @@
 import altair as alt
 import pandas as pd
+import numpy as np
+from app.vader_sentiment import vader_score, vader_compound_score
+from fastapi import FastAPI
+from app.data import MongoDB
+
+API = FastAPI(
+    title='Underdog Devs DS API',
+    version="0.46.2",
+    docs_url='/',
+)
+
+API.db = MongoDB("UnderdogDevs")
 
```

Comment:
Lines 8 through 14, the FastAPI code absolutely doesn't belong in this file. Come talk to me.